pkg/operator/options/options.go

```diff
@@ -98,7 +99,7 @@ func (o *Options) AddFlags(fs *FlagSet) {
 	fs.StringVar(&o.LogErrorOutputPaths, "log-error-output-paths", env.WithDefaultString("LOG_ERROR_OUTPUT_PATHS", "stderr"), "Optional comma separated paths for logging error output")
 	fs.DurationVar(&o.BatchMaxDuration, "batch-max-duration", env.WithDefaultDuration("BATCH_MAX_DURATION", 10*time.Second), "The maximum length of a batch window. The longer this is, the more pods we can consider for provisioning at one time which usually results in fewer but larger nodes.")
 	fs.DurationVar(&o.BatchIdleDuration, "batch-idle-duration", env.WithDefaultDuration("BATCH_IDLE_DURATION", time.Second), "The maximum amount of time with no new pending pods that if exceeded ends the current batching window. If pods arrive faster than this time, the batching window will be extended up to the maxDuration. If they arrive slower, the pods will be batched separately.")
-	fs.StringVar(&o.FeatureGates.inputStr, "feature-gates", env.WithDefaultString("FEATURE_GATES", "NodeRepair=false,SpotToSpotConsolidation=false"), "Optional features can be enabled / disabled using feature gates. Current options are: SpotToSpotConsolidation")
+	fs.StringVar(&o.FeatureGates.inputStr, "feature-gates", env.WithDefaultString("FEATURE_GATES", "ReservedCapacity=false,NodeRepair=false,SpotToSpotConsolidation=false"), "Optional features can be enabled / disabled using feature gates. Current options are: ReservedCapacity, NodeRepair, and SpotToSpotConsolidation")
```

Comment:
If we're updating the feature flag naming, let's update the design doc so that it reflects the same -- as it stands right now, that design doc says `CapacityReservations` is the feature flag

Reply:
I'm actually wondering if we just include the gate at the cloudprovider layer. This logic stays entirely dormant until a cloudprovider provides reserved offerings via the `GetInstanceTypes` implementation. I don't see a reason to add any gating in the upstream code, so I think we can drop the feature flag from here as well.

Reply:
Discussed offline: We need to keep this around because we can't re-define a flag in the cloudprovider repo